common/security_request_error.md

```diff
@@ -10,13 +10,20 @@ Understanding the `SecurityRequestError` exception.
 
 &nbsp;
 
+{: .note}
+> _For an understanding of IRRSMO00 return and reason codes, see the [IRRSMO00 Return and Reason Codes](https://www.ibm.com/docs/en/zos/3.2.0?topic=operations-return-reason-codes) documentation._

```

Comment:
This link seems to be broken.

Reply:
Other places this link is used should be checked too.

Reply:
Other instances of the link (index.md and downstream_fatal_error.md) are ok, looks like there was a typo while manually updating the "version number" in the doc here and not elsewhere. Good catch.